docs/user-guide/introspection-and-debugging.md

```diff
@@ -0,0 +1,296 @@
+
+To correct this situation, you can use ```kubectl stop rc``` to stop your Replication Controller and start a new one with fewer replicas, or you can use ```kubectl update``` to update your Replication Controller to 4 replicas, and then use ```kubectl stop pod``` to kill the pending pod. (If you don’t update the Replication Controller, it will re-create a pod after you kill the pending one.)
+
+In addition to ```kubectl describe pod```, another way to get extra information about a pod (beyond what is provided by ```kubectl get pod```) is to pass the ```-o yaml``` output format flag. This will give you even more information than ```kubectl describe pod```--essentially all of the information the system has about the Pod. Here you will see things like annotations (a kind of label for internal use only), restart policy, ports, and volumes.
+
+```yaml
+$ kubectl get pod my-nginx-i595c -o yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  annotations:
+    kubernetes.io/created-by: '{"kind":"SerializedReference","apiVersion":"v1","reference":{"kind":"ReplicationController","namespace":"default","name":"my-nginx","uid":"c555c14f-26d0-11e5-99cb-42010af00e4b","apiVersion":"v1","resourceVersion":"26174"}}'
+  creationTimestamp: 2015-07-10T06:56:21Z
+  generateName: my-nginx-
+  labels:
+    app: nginx
+  name: my-nginx-i595c
+  namespace: default
+  resourceVersion: "26243"
+  selfLink: /api/v1/namespaces/default/pods/my-nginx-i595c
+  uid: c558e44b-26d0-11e5-99cb-42010af00e4b
+spec:
+  containers:
+  - image: nginx
+    imagePullPolicy: IfNotPresent
+    name: nginx
+    ports:
+    - containerPort: 80
+      protocol: TCP
+    resources:
+      limits:
+        cpu: 600m
+        memory: 128Mi
+    terminationMessagePath: /dev/termination-log
+    volumeMounts:
+    - mountPath: /var/run/secrets/kubernetes.io/serviceaccount
+      name: default-token-zkhkk
+      readOnly: true
+  dnsPolicy: ClusterFirst
+  nodeName: kubernetes-minion-u619
+  restartPolicy: Always
+  serviceAccountName: default
+  volumes:
+  - name: default-token-zkhkk
+    secret:
+      secretName: default-token-zkhkk
+status:
+  conditions:
+  - status: "True"
+    type: Ready
+  containerStatuses:
+  - containerID: docker://9506ace0eb91fbc31aef1d249e0d1d6d6ef5ebafc60424319aad5b12e3a4e6a9
+    image: nginx
+    imageID: docker://319d2015d149943ff4d2a20ddea7d7e5ce06a64bbab1792334c0d3273bbbff1e
+    lastState: {}
+    name: nginx
+    ready: true
+    restartCount: 0
+    state:
+      running:
+        startedAt: 2015-07-10T06:56:28Z
+  hostIP: 10.240.112.234
+  phase: Running
+  podIP: 10.244.3.4
+  startTime: 2015-07-10T06:56:21Z
+```
+
+## Example: debugging a down/unreachable node
```

Comment:
Hm, this seems like an admin task, not a user task. We could have this somewhere in the admin guides and link there from here?